news/2021-09-18-200th-featured-artist-our-stolen-theory.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "200th Featured Artist: Our Stolen Theory"
+date: 2021-09-18 09:30:00 +0000
+---
+
+We're celebrating a massive milestone with the official inclusion of **Our Stolen Theory** - United (L.A.O.S Remix)** in the Featured Artist program!
+
+![](https://assets.ppy.sh/artists/200/header.jpg)
+
+Every osu! player recognizes the modern electronic pop/rock sound behind [**Our Stolen Theory - United (L.A.O.S Remix)**](https://osu.ppy.sh/beatmapsets?q=artist%3D%22our%20stolen%20theory%22%20united).
+
+This song has penetrated every corner of the osu! community, with ranked beatmaps spanning all game modes and totaling nearly 8 million plays across the board.
+
+Unlike our usual artist announcements, there's only 1 song available on [**Our Stolen Theory**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/200). The band hasn't actively produced music in a long time, but their track *United (L.A.O.S Remix)* has made an undeniable impact on the osu! community, and we're beyond happy to carve out a special place for it in the osu! community.
+
+---
+
+As new osu! players climb ranks and older ones retire, some major events in the community fade from memory.
+
+Even so, [Azer](https://osu.ppy.sh/users/2155578)'s journey to full combo-ing [the first ranked map of *United (L.A.O.S Remix)*](https://osu.ppy.sh/beatmapsets/237768) with HDHR is an event that'll never be forgotten.
+
+Regardless of whether you're a new player or an osu! veteran, relive the excitement of this play through Azer's video, then play through any of the other awesome beatmaps of listed below!
+
+<iframe width="100%" height="400" src="https://www.youtube.com/embed/VkjtPz47UQs" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Our Stolen Theory - United (L.A.O.S Remix)
+
+Experience this timeless classic for yourself through any of these top-tier beatmaps:
+
+- [osu! map](https://osu.ppy.sh/beatmapsets/237768) by [Asphyxia](https://osu.ppy.sh/users/1715720) - the first ranked and most popular of the bunch!
+- [osu! map](https://osu.ppy.sh/beatmapsets/953303) by [Sotarks](https://osu.ppy.sh/users/4452992) - a modern rendition showcasing a different style!
+- [osu!taiko map](https://osu.ppy.sh/beatmapsets/559371) by [hikiko-](https://osu.ppy.sh/users/6512678)
+- [another osu!taiko map](https://osu.ppy.sh/beatmapsets/556731) by [Yuzeyun](https://osu.ppy.sh/users/481582)
+- [osu!catch map](https://osu.ppy.sh/beatmapsets/527777) - loved beatmap featuring an outstanding storyboard!
+- [osu!mania map](https://osu.ppy.sh/beatmapsets/726545) - a full spread for 4K players!
```

Comment:
o just realized this as well

for some reason the mapper name on the last two entries here is missing 😦